examples/decomposition/plot_ica_vs_pca.py

```diff
@@ -57,7 +57,7 @@
 S_ica_ /= S_ica_.std(axis=0)
```

Comment:
This cell currently emits a warning that could be silenced by altering line 54 to add the argument `whiten="unit-variance"` as follows:

```diff
-ica = FastICA(random_state=rng)
+ica = FastICA(random_state=rng, whiten="unit-variance")
```

Reply:
Thanks @jsilke! It seems to me that the current default of `whiten` is `arbitrary-variance`. What about adding the argument 'whiten="arbitrary-variance', which can also silence the warning. Please correct me if I was wrong.

Reply:
Your suggestion to use `whiten="arbitrary-variance"` to silence the warning sounds reasonable and preserves the current behavior. I would be happy with this as well!

Reply:
Thanks again @jsilke. I have adopted `whiten="arbitrary-variance"` to the code. Please have a check.